frontend/spa/src/components/ImageLabeling.vue

```diff
@@ -660,25 +632,73 @@ export default {
             this.clear_canvas_event = !this.clear_canvas_event;
             console.log('clear canvas called', this.clear_canvas_event)
         },
+        
         changeRadio: function(){
             this.change_radio_event = !this.change_radio_event;
         },
 
+        filtered: function(){
+            this.save_progress();
+            this.$store.dispatch('image_labeling/change_filter');
+            this.set_first_image()
+        },
+        
+        defaultDrawTools: function() {
+            this.tools_list = {};
+            this.tools_list["freehand"] = "Freehand";
+            this.tools_list["polygon"] = "Polygon";
+            this.tools_list["rectangle"] = "Rectangle";
+            this.tools_list["point"] = "Point";
+            this.tools_list["circle"] = "Circle";
+        },
+        
+        disableDrawTools: function() {
```

Comment:
The defaultDrawTools and disableDrawTools function is missing the select option. Can we add this back as well?